src/app/components/account-settings/account-settings.component.ts

```diff
@@ -36,6 +36,10 @@ import { AppStateService } from '@app/services/app-state.service';
                     <mat-icon>edit_note</mat-icon>
                     <span>Signing</span>
                 </button>
+                <button mat-menu-item (click)="navigateToSignMessage()">
```

Comment:
I have some questions around why this logic broken out into it's own page when there's already a page dedicated for signing.  Seems kind of weird to split this out into a separate route & a different menu navigation item.

Reply:
Menu item removed